wg-ai-conformance/README.md

```diff
@@ -19,7 +19,7 @@ The [charter](charter.md) defines the scope and governance of the AI Conformance
 ## Meetings
 *Joining the [mailing list](https://groups.google.com/a/kubernetes.io/g/wg-ai-conformance) for the group will typically add invites for the following meetings to your calendar.*
 * Regular WG Meeting ([calendar](https://calendar.google.com/calendar/embed?src=a574542f85d51cb4624145f13307810c9e23bc9096cfbb83f6205cc7acee34b6%40group.calendar.google.com)): [Thursdays at 10:00 PT (Pacific Time)](https://zoom.us/j/95480364337) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10%3A00&tz=PT%20%28Pacific%20Time%29).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1qlW1LkibOoiMio-hbJucRjOYeKT8mNaQ4awwzf0bi8M/edit).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1qlW1LkibOoiMio-hbJucRjOYeKT8mNaQ4awwzf0bi8M/edit). (Join mailing list for access)
```

Comment:
I've updated the permission so that it's viewable to the public, without joining the mailing list. 

This md file is generated, so you need to update sigs.yaml, then run `make`, and then
commit your changes to sigs.yaml and any generated docs.